debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -573,6 +573,14 @@ public static OversizeHandlingMode parse(String value, String defaultValue) {
             .withDescription("Internal use only")
             .withType(Type.LIST);
 
+    /**
+     * The {@link ReplicaSets#SEPARATOR}-separated list of connection strings
+     */
+    public static final Field ALLOW_OFFSET_INVALIDATION = Field.createInternal("mongodb.internal.allow.offset.invalidation")
```

Comment:
```suggestion
    public static final Field ALLOW_OFFSET_INVALIDATION = Field.createInternal("mongodb.allow.offset.invalidation")
```
It is already prefixed with `internal`

Reply:
Hm... Then I think the following is not exactly correct either

```
public static final Field TASK_CONNECTION_STRINGS = Field.createInternal("mongodb.internal.task.connection.strings")
            .withDescription("Internal use only")
            .withType(Type.LIST);
```

BTW  wont the change result in the filed named `internal.mongodb.allow.offset.invalidation` then?

Reply:
`TASK_CONNECTION_STRINGS` is however truly internal and is of no concern to the user (as it is internally set). Additionally it's gone in 2.6 so we can safely ignore that :)